File: Palindrome Number/PalindromeNumber.java
class Solution {
    public boolean isPalindrome(int x) {
        String str = String.valueOf(x);
        char array[] = str.toCharArray();
        String a[] = new String[array.length];
        for (int i = 0, j = array.length - 1; i < array.length && j >= 0; i++, j--) {
            a[i] = String.valueOf(array[j]);
        }
        String reverse = String.join("",a);
        if (!reverse.equals(String.valueOf(x)))  return false;
        return true;
    }
}

File: 9. Palindrome Number/PalindromeNumber.js
/**
 * @param {number} x
 * @return {boolean}
 */
var isPalindrome = function(x) {
    var a = String(x);
    var b = String(a.split('').reverse().join(""));
    if (a == b) {
        return true;
    }
    return false;
};